http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -8,29 +8,21 @@ set -e
 # 'debconf: unable to initialize frontend: Dialog'
 export DEBIAN_FRONTEND=noninteractive
 
-#create user to run jupyter
-VM_JUP_USER=jupyter
+##### JUPYTER USER SETUP #####
+# Create the jupyter user that corresponds to the jupyter user in the jupyter container
+VM_JUP_USER=jupyter-user
+VM_JUP_USER_UID=1002
 
-sudo useradd -m -c "Jupyter User" $VM_JUP_USER
+sudo useradd -m -c "Jupyter User" -u $VM_JUP_USER_UID $VM_JUP_USER
 sudo usermod -a -G $VM_JUP_USER,adm,dialout,cdrom,floppy,audio,dip,video,plugdev,lxd,netdev $VM_JUP_USER
 
-## Change ownership for the new user
-
-sudo chgrp $VM_JUP_USER /anaconda/bin/*
-
-sudo chown $VM_JUP_USER /anaconda/bin/*
-
-sudo chgrp $VM_JUP_USER /anaconda/envs/py38_default/bin/*
-
-sudo chown $VM_JUP_USER /anaconda/envs/py38_default/bin/*
-
-sudo systemctl disable --now jupyterhub.service
-
 
+##### PD MOUNTING #####
 # Formatting and mounting persistent disk
-WORK_DIRECTORY="/home/$VM_JUP_USER/persistent_disk"
-## Create the PD working directory
-mkdir -p ${WORK_DIRECTORY}
+# Note that we cannot mount in /mnt/disks/work as it is a temporary disk on the DSVM!
+PD_DIRECTORY="/home/$VM_JUP_USER/persistent_disk"
+## Create the working and persistent disk directories
+mkdir -p ${PD_DIRECTORY}
```

Comment:
is this comment accurate, isn't it just making the PD directory?

Reply:
Yep! Should be fixed now